docs/node/RPC Category/16-loan.md

```diff
@@ -330,6 +330,48 @@ interface loan {
 }
 
 interface VaultDetails {
+  vaultId?: string
+  loanSchemeId: string
+  ownerAddress: string
+  isUnderLiquidation: boolean
+  batches?: AuctionBatchDetails[]
+  collateralAmounts?: string[]
+  loanAmount?: string[]
+  collateralValue?: BigNumber
+  loanValue?: BigNumber
+  currentRatio?: BigNumber
+}
+
+interface AuctionBatchDetails {
+  index: BigNumber
+  collaterals: string[]
+  loan: string
+}
+```
+
+## listVaults
+
+List all available vaults.
+
+```ts title="client.loan.listVaults()"
+interface loan {
+  listVaults (options: ListVaultOptions, pagination: VaultPagination): Promise<VaultDetails[]>
```

Comment:
listVaults (pagination: VaultPagination = {}, options: ListVaultOptions = {}): Promise<VaultDetails[]>

Like what you wrote in the js file